src/pyats/contrib/plugins/webex.py

```diff
@@ -94,7 +98,46 @@ def post_job(self, job):
                         'notification will be sent')
             return
 
-        payload = {'markdown': MESSAGE_TEMPLATE.format(job=job)}
+        # Format message with info from job run
+        msg = MESSAGE_TEMPLATE.format(job=job)
+
+        try:
+            # Attempt to get path for TRADe logs
+            if not self.runtime.runinfo.no_upload:
+                msg += '\n\nView TRADe logs at: %s'\
+                       % self.runtime.runinfo.log_url
+        except AttributeError:
+            pass
+
+        try:
+            host = job.runtime.env['host']['name']
+            # Determine if liveview is running
+            if self.runtime.args.liveview and\
+                    self.runtime.args.liveview_keepalive:
+                # Liveview will set this to the assigned port if not specified
+                port = self.runtime.args.liveview_port
+                try:
+                    # Attempt to add a link using the host domain name
+                    addr = socket.getfqdn()
+                    socket.gethostbyname(addr)
```

Comment:
What's the point of this line?

Reply:
This is a check that the address is resolvable. `socket.getfqdn()` will return the hostname even if that isn't a domain on the network.